src/core/engines/cross-margin/CrossMarginMath.sol

```diff
@@ -69,7 +70,7 @@ library CrossMarginMath {
             CrossMarginDetail memory detail = details[i];
 
             if (detail.callWeights.length != 0 || detail.putWeights.length != 0) {
-                (int256 cashCollateralNeeded, int256 underlyingNeeded) = getMinCollateral(detail);
+                (int256 cashCollateralNeeded, int256 underlyingNeeded) = _getMinCollateralWithoutVerification(detail);
```

Comment:
We still want verification

Reply:
I'll update this after merging in

Reply:
the details you get from `_getAccountDetails` should not contain any 0 amounts

Reply:
oh maybe we skip the 0 amount check but still need to length check?